client/src/components/PredictionPanel.vue

```diff
@@ -86,7 +86,7 @@ export default defineComponent({
       }
       return null;
     },
-    pwsId(): string | null {
+    pwsId(): BoundedGeoDatum | null {
```

Comment:
I'm unsure if I've changed this in all the right places. Apologies in advance if this is sheer butchery.

Reply:
The client should fail to compile if you had missed something, IIUC.

Reply:
Perfect. Makes me feel better about it 😅

Reply:
Gotcha. Makes me feel a little better 😅